components/collector/src/source_collectors/trivy/security_warnings.py

```diff
@@ -26,14 +26,24 @@ class TrivyJSONVulnerability(TypedDict):
     References: list[str]
 
 
-class TrivyJSONDependencyRepository(TypedDict):
+class TrivyJSONResult(TypedDict):
     """Trivy JSON for one dependency repository."""
 
     Target: str
     Vulnerabilities: list[TrivyJSONVulnerability] | None  # The examples in the Trivy docs show this key can be null
 
 
-TrivyJSON = list[TrivyJSONDependencyRepository]
+TriviJSONSchemaVersion1 = list[TrivyJSONResult]
+
+
+class TrivyJSONSchemaVersion2(TypedDict):
+    """Trivy JSON conform schema version 2, see https://github.com/aquasecurity/trivy/discussions/1050."""
```

Comment:
Is this still accurate, does it exist? The [docs regarding container scanning](https://aquasecurity.github.io/trivy/v0.55/docs/target/container_image/) don't name a Trivy JSON format.
In fact, [section SBOM > Generation detail](https://aquasecurity.github.io/trivy/v0.55/docs/supply-chain/sbom/) states that CycloneDX or SPDX are the only two (SBOM) formats.
Note that Trivy version 0.55.2 was only released yesterday :D I think it'd be good to at least find docs/spec.
After digging through the docs for a while, it seems that the format is listed under [Reporting > JSON](https://aquasecurity.github.io/trivy/v0.55/docs/configuration/reporting/#json).
Should we really support this custom Trivy format, when it also offers the SARIF and SBOM formats?

Reply:
I've opened a discussion at https://github.com/aquasecurity/trivy/discussions/7552. The response was to open a ticket to improve the docs, so apparently the format is still supported.